Jenkinsfile

```diff
@@ -512,5 +512,19 @@ def boolean skipWorkerBuild() {
         return true
     }
 
+    if (sh(
+        returnStatus: true,
+        script: "git diff --name-only origin/${env.CHANGE_TARGET}... | grep -v '^\\.'"
```

Comment:
Now this rule has been added, can the previous .md in line 496 and .github in line 510 be removed?

Reply:
I think they serve different purpose, particularly the one that is matching for all *.md files (while the added one is for hidden files from the root folder only).